golden-tests/conditions.dhall

```diff
@@ -1,45 +1,53 @@
-    let OS = constructors ./dhall/types/OS.dhall 
+    let stdlib = ./dhall/stdlib.dhall 
+
+in  let OS = constructors ./dhall/types/OS.dhall 
```

Comment:
They actually already are, I just forgot :) It's in `stdlib.`constructors``. I didn't like types because they aren't types, but constructors needs quoting. So I do need a better name, maybe `types` is good enough.